GoMeet/src/main/webapp/js/create-meeting-step1.js

```diff
@@ -0,0 +1,41 @@
+let guestsAdded = 0; // Number of guest email inputs on the page 
+
+/**
+ * Add an input element to the guest list for user to add email 
+ * @param {*} doc the document object to be manipulated 
+ */
+function addGuest(doc) {
+  guestsAdded += 1; 
+  let listItem = doc.createElement('li');
+  let guestNumber = guestsAdded; 
+  listItem.id = 'guest-' + guestNumber; 
+
+  let newTextInput = doc.createElement('input'); 
+  newTextInput.type = 'text';
+  newTextInput.name = 'guest-' + guestNumber
+
+  let newInputLabel = doc.createElement('label');
+  newInputLabel.htmlFor = 'guest-' + guestNumber;
+  newInputLabel.innerText = 'Guest: ';
+
+  // delete button associated with this guest
+  let removeGuestButton = doc.createElement('button');
+  removeGuestButton.innerHTML = 'X';
```

Comment:
Prefer using innerText here as this is just the text of the element.

Reply:
Fixed :)